lms/static/js/spec/student_account/helpers.js

```diff
@@ -65,6 +65,23 @@ define(['underscore'], function(_) {
     };
     var IMAGE_MAX_BYTES = 1024 * 1024;
     var IMAGE_MIN_BYTES = 100;
+    var SOCIAL_PLATFORMS = {
+        facebook: {
+            display_name: 'Facebook',
+            url_stub: 'facebook.com/',
+            example: 'https://www.facebook.com/edX'
```

Comment:
These are showing up as the default text in the UI.  Don't we want these to match the others and have /username?